include/deal.II/opencascade/utilities.h

```diff
@@ -467,7 +467,6 @@ namespace OpenCASCADE
 
 DEAL_II_NAMESPACE_CLOSE
 
-#  endif // DEAL_II_WITH_OPENCASCADE
+#endif // DEAL_II_WITH_OPENCASCADE
 
 #endif // dealii_occ_utilities_h
```

Comment:
```suggestion
#endif
```

Pedantic me found one last filename here, but this one obviously does not affect the indentation. You got rid of these filenames in other occasions.

Reply:
I have removed that. I was looking for comments with file names `grep -r -e "/\*-----------" | grep ".h -". The problem is the format of the comment.